### ISSUE TITLE
Use gender neutral language on landing page.

### ISSUE THREAD

**Issue by yahelc** [scoutapp/scout_realtime#6]
The copy on <a href="https://github.com/scoutapp/scout_realtime/blob/gh-pages/index.html#L47
">`index.html`</a> on the  `gh-pages` branch currently reads:

```
30 years ago, top was born. scout_realtime is top for the modern man.
```

This is a great tool, but there's nothing gender specific about its usage. It would be great to change it to something less exclusionary, like `dev` or `sysadmin`. 

I'm happy to submit a pull request with a change.

**Comment by gdiggs** [comment]
I submitted this: #7 

I think saying something like "modern developer" still exhibits the idiom while being inclusive.

**Comment by yahelc** [comment]
This was fixed by @andre in 78bb605.
